ecpy_hqc_legacy/tasks/tasks/instr/views/spdev_views.enaml

```diff
@@ -69,6 +69,17 @@ enamldef DemodSpView(InstrView): view:
                     "then average.\nThis allows to use a ref Iref, Qref "
                     "and correct each I,Q pair by the ref, then average.")
 
+    Label: num_loop:
+        text = 'Number of pulse sequences in loop'
+    QtLineCompleter: num_loop_val:
+        text := task.num_loop
+        entries_updater << task.list_accessible_database_entries
+        tool_tip = ('Number of loops in your pulse sequence \n '
```

Comment:
You can append the standard tool tip for completion after your custom one.

Reply:
ok